bindings/c/test/mako/mako.h

```diff
@@ -171,14 +175,15 @@ typedef struct {
 typedef struct {
 	int worker_id;
 	pid_t parent_id;
-	FDBDatabase* database;
 	mako_args_t* args;
 	mako_shmhdr_t* shm;
+	FDBDatabase* databases[10];
```

Comment:
I hard-coded this to 10 only for proof of concept / it's sufficient for current testing scenarios.

Reply:
Define a macro? ... but, this, I think it's a good idea to make this dynamic even in PoC. What if we want to test 16? then 32?